src/contexts/UserContext.js

```diff
@@ -31,6 +31,15 @@ export const UserContextProvider = ({ children }) => {
     );
 
     setChatClient(client);
+
+    if (Number(client.userID) !== 28) {
+      console.log(client);
+      const channel = client.channel("messaging", {
+        members: [client.userID, "28"],
+      });
+      // Here, 'travel' will be the channel ID
+      await channel.create();
```

Comment:
Is this the collectify admin chat?

Reply:
Yup, I'll change it to a variable and add some comments so it's clearer